app/Models/MiscModel.php

```diff
@@ -126,12 +126,13 @@ public function getLink(string $action = 'show'): string
     /**
      * Create the model's Entity
      */
-    public function createEntity(): Entity
+    public function createEntity(string $newEntry = ''): Entity
```

Comment:
Entities can have a NULL entry (for example those created in the API), which _might_ cause this call to crash?